wallet_dht.py

```
@@ -0,0 +1,205 @@
+
+    async def _update_wallet_directory(self):
+        """Update the directory of all wallets in the network"""
+        try:
+            # Get existing wallet directory
+            wallets_key = "global:wallets"
+            wallets_data = await self.dht.get(wallets_key)
+
+            if wallets_data:
+                wallets = json.loads(wallets_data)
+            else:
+                wallets = []
```

Ruff (0.8.2)

Use ternary operator `wallets = json.loads(wallets_data) if wallets_data else []` instead of `if`-`else`-block

Replace `if`-`else`-block with `wallets = json.loads(wallets_data) if wallets_data else []`

(SIM108)